services/app-web/src/hooks/useTealium.ts

```diff
@@ -29,8 +29,8 @@ const useTealium = (): {
     // Add Tealium setup
     // this effect should only fire on initial app load
     useEffect(() => {
-        // Do not add tealium for local dev
-        if (process.env.REACT_APP_STAGE_NAME === 'local') {
+        // Do not add tealium for local dev or review apps
+        if (process.env.REACT_APP_AUTH_MODE !== 'IDM') {
```

Comment:
The tealium changes in this PR are due to realizing that on review apps (which have a random domain) Tealium tags will not load properly because it does not recognize the URL. This causes unnecessary API calls and React updates because we keep trying to find tealium and fail.